### DISCUSSION TITLE
Changing the design of card

### DISCUSSION THREAD

**Discussion by AdamRamsi** [kormax/apple-home-key-reader#22] [Q&A]
Hi, 

First of all I want to thank you for such a perfect work. I have a question about the design of the card added to Apple Wallet (For example: image, logo, different color of the background, etc.).  In which part can I implement it?  

    "homekey": {
        "persist": "homekey.json",
        "express": true,
        "finish": "black",
        "flow": "fast"
        
In the "entity.py" and "configuration.json" file, I found that you can vary between 4 colors. But I can't find where and how to add a picture to the background for example. I would be very grateful for your help.

**Comment by kormax** [comment]
You can't.

The "card" is generated by Apple and they make a decision on what art to display there.

It just so happens that they added 4 variants based on lock color, which we can influence.
